apps/videos-admin/src/components/FormSelectField/FormSelectField.stories.tsx

```
@@ -0,0 +1,86 @@
+    }
+  }
+}
+
+export const Helper: Story = {
+  args: {
+    ...Filled.args,
+    helperText: 'Members of the Fellowship'
+  }
+}
+
+export const Error: Story = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)